app/views/shared/dialogs/_dialog_field.html.haml

```diff
@@ -31,7 +31,15 @@
       = render(:partial => "shared/dialogs/dialog_field_radio_button", :locals => locals)
 
     - when "DialogFieldDropDownList"
-      = render(:partial => "shared/dialogs/dialog_field_drop_down_list", :locals => locals)
+      - category_tags = field.values.collect(&:reverse)
```

Comment:
`category_tags` is probably not an accurate description of the values being collected here.

Also, is there a reason we're blindly reversing the collection? Shouldn't `field.values` already return the values in the correct order based on what the `sort_order` is set to?